Form/Type/DateTimeRangeType.php

```diff
@@ -24,11 +24,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
     public function configureOptions(OptionsResolver $resolver)
     {
-        $resolver->setDefaults(
-            [
-                'field_type' => DateTimeType::class,
-                'field_options' => [],
-            ]
-        );
+        $resolver->setDefaults([
+            'field_type' => DateTimeType::class,
+            'field_options' => ['html5' => false, 'format' => 'non-html5'],
```

Comment:
proc chceme globalne vynucovat nastaveni pro `html5` nemelo by spis byt reseno na urovni projektu?
nechapu jak funguje `non-html5` format, podle me je to nevalidni hodnota s kterou to nefunguje, muzes prosim vysvetlit?

dekuju.

Reply:
@ldfk je to spíše workaround, ale zatím jsem nenašel lepší způsob, jak se zbavit ```Cannot use the "date_format" option of the "Symfony\Component\Form\Extension\Core\Type\DateTimeType" with an HTML5 date.```. Hodnota `format` by se měla lišit od `DateTimeType::HTML5_FORMAT` (`yyyy-MM-dd'T'HH:mm:ss`). Ale dá se dát něco správnějšího jako `yyyy-MM-dd`.
Nebo máš lepší nápad?

Reply:
1. tohle je o nastaveni konktertniho projektu a ne bundlu. tohle z bundlu prosim smaz a nastav si to jak potrebujes pro dany projekt pomoci [FormTypeExtension](https://symfony.com/doc/current/form/create_form_type_extension.html)
2. `non-html5` je nesmysl a pokud by jsi nacetl formular s nejakymi (default) hodnotami, tak se s pouzitim `non-html5` zadne datum do formulare nenacte a mel by jsi tedy pouzit nejaky validni format.